README.md

```diff
@@ -24,11 +24,14 @@ crosswords and the regular ones.
 git clone https://github.com/idoneam/minicrossword-bot.git
 cd minicrossword-bot
 
-# Set up a virtualenv named `env`
-virtualenv env   
+# Set up a virtualenv named `env` (replace python3 with python on Windows)
```

Comment:
```suggestion
# Set up a virtualenv named `env` (replace python3 with python on Windows)
```

Is this really OS-specific? I just use `python` on macOS.

Reply:
on windows, `python3` doesn't point to anything, so it requires that change, but on all other OSes, `python3` points to the default `python3.x` install, even if `python` is also made to point to `python3.x`
this does mean that `pyenv` users are going to be typing an extra 3, but adding that caveat seems overlong for a simple guide instruction

Reply:
I'd just keep it simple and not add a note at all because we specify a Python 3.7 minimum in our README anyways. It's up to the developer to have Python set up correctly as a pre-req